src/Home_Page.html

```diff
@@ -10,28 +10,13 @@
           integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 <body style="background-color:whitesmoke; font-family:Helvetica Neue,Helvetica,Arial,sans-serif;">
-
-
-
 <form action=form/submit" method="GET" style="align-items: center; margin: auto; color: red; width:30%;">
-
-
     <div class="form-group card text-center">
         <label for="txtSearch"></label>
         <input type="text" class="form-control" name="SearchText" id="txtSearch" aria-describedby="helpId" placeholder="Search for a topic">
         <input name="SearchButton" id="btnSearch" class="btn btn-primary" type="submit" value="Search">
     </div>
-
-
-
 </form>
-
-
-
-
-
-
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure